docs/DeveloperGuide.adoc

```diff
@@ -238,6 +238,7 @@ The undo/redo mechanism is facilitated by an `UndoRedoStack`, which resides insi
 
 `UndoRedoStack` only deals with `UndoableCommands`. Commands that cannot be undone will inherit from `Command` instead. The following diagram shows the inheritance diagram for commands:
 
+.Command Class Diagram
```

Comment:
the in-built captions aren't that great when it comes to formatting because of line-spacing issues. Could you change it to _Figure {Number}. Command Class Diagram_? This is better in terms of formatting, but the only drawback is that you'll have to manually add the figure numbers :/

Reply:
Sure